components/button/button.css

```diff
@@ -0,0 +1,134 @@
+@import url("/components/root.css");
+.ct-btn {
+  border: none;
+  padding: 0.5rem 1.2rem;
+  cursor: pointer;
+  background: var(--body);
+  border-radius: 3px;
+  display: flex;
+  align-items: center;
+}
+
+/* Default buttons styles  */
+.ct-btn.ct-red {
+  background: var(--red-500);
+  color: var(--body);
+  transition: all 250ms ease;
+}
+.ct-btn.ct-red:hover {
+  background: var(--red-700);
+  color: #fff;
+}
+.ct-btn.ct-blue {
+  background: var(--blue-500);
+  color: var(--body);
+  transition: all 250ms ease;
+}
+.ct-btn.ct-blue:hover {
+  background: var(--blue-700);
+  color: #fff;
+}
+.ct-btn.ct-green {
+  background: var(--green-500);
+  color: var(--body);
+  transition: all 250ms ease;
+}
+.ct-btn.ct-green:hover {
+  background: var(--green-700);
+  color: #fff;
```

Comment:
color variable may be?

Reply:
Actully I explicitly want that white . color but I'll look into it or create new var for white I think. Thanks for that too